shipyard-controller/debug-ui/fetch_blockingsequences.js

```diff
@@ -0,0 +1,37 @@
+function fetchBlockingSequences(
+  project,
+  context,
+  stage,
+  blockingSequencesList
```

Comment:
I would suggest a better naming to suggest that this parameter holds the HTML DOM element where the results are stored. Something like:
```suggestion
  HTMLItem_blockingSequencesList
```
```suggestion
  targetHTML
```